File: 02. CONV_UNI_SOAP_DOTNET_GUARDERAS/01. SERVIDOR/WS_CONVUNI_SOAPDOTNET/CONVUNI/WS_CONVUNI_SOAPDOTNET/wc.edu.monster.prueba/Prueba01.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using WS_CONVUNI_SOAPDOTNET.ec.edu.monster.controlador;

namespace WS_CONVUNI_SOAPDOTNET.ec.edu.monster.prueba
{
    public class Prueba01
    {

        public static void Main(string[] args)
        {
            double c = 0;
            double f = 32;

            ConvuniControlador controlador = new ConvuniControlador();
            object nuevF = controlador.convCF(c);
            object nuevC = controlador.convFC(f);

            Console.WriteLine(c + "°C = " + nuevF + "°F");
            Console.WriteLine(f + "°F = " + nuevC + "°C");
        }
    }
}

File: 02. CONV_UNI_SOAP_DOTNET_GUARDERAS/02. CLIWEB/WS_CONVUNI_SOAPDOTNET_CLIWEB/ec/edu/monster/prueba/prueba01.py
from ec.edu.monster.controlador.controladorServicio import ControladorServicio

class Prueba01:
    
    def probar(self):
        c = 0
        f = 32
        
        conServicio = ControladorServicio()
        
        nuevF = conServicio.convCF(c)
        nuevC = conServicio.convFC(f)
        
        print(str(c)+"°C = "+str(nuevF)+"°F")
        print(str(f)+"°F = "+str(nuevC)+"°C")
        
        
if __name__ == "__main__":
    pr1 = Prueba01()
    pr1.probar()